### PR TITLE
[new] MST integrations

### PR DESCRIPTION
**Pull request by Rajakavitha1**
# Description

_ converted Monitor MST with Prometheus to how-to.
_ grouped all the third party integrations with MST under 'Integrations'

# Links

Fixes https://github.com/timescale/docs/issues/1185

# Writing help

For information about style and word usage, see the [style guide](https://docs.timescale.com/timescaledb/latest/contribute-to-docs/)

# Review checklists
Reviewers: use this section to ensure you have checked everything before approving this PR:
@avthars  Pleaser review the content for technical accuracy because there has been a lot of changes in the UI for integrations in MST since when the tutorial was written.

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: mst/integrations/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+title: Integrations for Managed Service for TimescaleDB
+excerpt: A Command Line tool for fully managed TimescaleDB service on AWS, Azure, or GCP.
+product: mst
+---
+
+# Integrations for Managed Service for TimescaleDB
+
+Managed Service for TimescaleDB integrates with the other tools you are already
+using. You can integrate one Managed Service for TimescaleDB service with one or
+more Managed Services. You can combine your Managed Service for TimescaleDB
```
Comment (Loquacity) [inline mst/integrations/index.md:11]: This sentence is a little confusing (maybe it's the repetition?)

File: mst/integrations/index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+services with third party tools and build a truly cloud data platform.
```
Comment (Loquacity) [inline mst/integrations/index.md:12]: I think there a word missing here ...

File: mst/integrations/index.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+You can integrate Managed Service for TimescaleDB with:
+
+*   Grafana
+*   Syslog
+*   Datadog
+*   Prometheus
+*   External Elasticsearch
+*   External OpenSearch
```
Comment (Loquacity) [inline mst/integrations/index.md:21]: Can we make these links?

File: mst/page-index/page-index.js [hunk 4]
```diff
@@ -38,21 +38,37 @@ module.exports = [
...
-      },
-      {
-        title: "Integrate Managed Service for TimescaleDB and Google Data Studio",
-        href: "google-data-studio-mst",
+        title: "Integrations",
+        href: "integrations",
+        excerpt: " MST integrates with the other tools you are already using, and makes it easy to add more integrations",
+        children:  [
+          {
+           title: "Integrate Managed Service for TimescaleDB and Google Data Studio",
```
Comment (Loquacity) [inline mst/page-index/page-index.js:51]: Let's change these titles to be parallel with each other. Might be best to use "verbing with product" as a structure, so we can have "Visualizing data with Grafana" and "Visualizing data with Google Data Studio" (or whatever).

File: mst/integrations/prometheus-mst.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
+---
+title: Prometheus endpoint for Managed Service for TimescaleDB
+excerpt: Use Prometheus to monitor your Managed Service for TimescaleDB
+product: mst
+keywords: [Prometheus, monitor, integration]
+---
+
+# Set up a Prometheus endpoint for a Managed TimescaleDB database
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply mst/integrations/prometheus-mst.md:8]: Agreed

**Comment by Loquacity** [comment]
@Rajakavitha1 What's the status of this one?

**Comment by Loquacity** [comment]
@Rajakavitha1 Do you still want to merge this one? I recommend getting it ready and merging before we merge any cloud-centric stuff, or it's going to be even more of a mess of merge conflicts than it is already ...

**Comment by Rajakavitha1** [comment]
> @Rajakavitha1 Do you still want to merge this one? I recommend getting it ready and merging before we merge any cloud-centric stuff, or it's going to be even more of a mess of merge conflicts than it is already ...

Yes @Lana let me work on this and take it to merge and yes!!! it will only add to merge conflicts later...

**Pull request closed without merging**